### ISSUE TITLE
Fine-tuning on VTAB-1k.

### ISSUE THREAD

**Issue by Yewen1486** [AFeng-x/HST#3]
Thanks for you great work! I want to reproduce results on VTAB-1k. I noticed code in 'https://github.com/AFeng-x/HST/blob/main/data/dataset_factory.py' L169-170 is commented out. Should I use VTAB(root=root, train=is_training, **kwargs) to build the dataset?

**Comment by AFeng-x** [comment]
Yes, you could also follow the code in "https://github.com/dongzelian/SSF/tree/main/data" to get more information to build the VTAB dataset.
